app/components/MixerVolmeter.vue.ts

```diff
@@ -55,7 +53,7 @@ export default class MixerVolmeter extends Vue {
   }
 
   get backgroundColor() {
-    return this.customizationService.nightMode ? NIGHT_BG : DAY_BG;
+    return this.customizationService.getThemeBackground();
```

Comment:
This will probably only ever be mounted in the main window, and it will be reactive there.  But probably best to make this a getter so it's reactive anywhere.